### PR TITLE
Adds eslint to dependencies and resolves code to match rules

### PR DESCRIPTION
**Pull request by jejacks0n**
Add eslint to the dev dependencies for create-freesewing-pattern templates. When firing up a new pattern I was getting some eslint errors/warnings -- because the dependencies weren't clear, and so I've added the relevant plugins and standard configurations.

After installing the eslint dependencies, I resolved any violations that I found.

### CODE REVIEW

File: packages/create-freesewing-pattern/template/freesewing/example/src/App.js [hunk 1]
```diff
@@ -26,17 +25,19 @@ const App = (props) => {
...
+  } else {
+    recreate = false
+  }
 
   return (
     <Workbench
       freesewing={freesewing}
       Pattern={Pattern}
-      userLanguage="{{language}}"
+      userLanguage="en"
```
Comment (jejacks0n) [inline packages/create-freesewing-pattern/template/freesewing/example/src/App.js:38]: Just noticing this might be inadvertently changed. Needs to be resolved before merging.

Comment (joostdecock) [reply packages/create-freesewing-pattern/template/freesewing/example/src/App.js]: Put this back to be the placeholder

**Comment by joostdecock** [comment]
Hey, this is great stuff. Just one thought on the matter: We use prettier across (certain parts of) the monorepo to keep consistent code formatting.

Obviously, a linter is more/better, we just need to keep an eye on things so that we don't end up in this perpetual back and forth between eslint and prettier changing things like trailing commas and whatnot.

Not saying this will happen. Just writing this down so when it does, I'll remember why ;)

**Comment by jejacks0n** [comment]
> Hey, this is great stuff. Just one thought on the matter: We use prettier across (certain parts of) the monorepo to keep consistent code formatting.
> 
> Obviously, a linter is more/better, we just need to keep an eye on things so that we don't end up in this perpetual back and forth between eslint and prettier changing things like trailing commas and whatnot.
> 
> Not saying this will happen. Just writing this down so when it does, I'll remember why ;)

Haha, yeah, I saw that but didn’t do a deep dive to find/resolve differences. I think we should change the linting rules to match prettier if that’s the case. I’m not opinionated on the rules, just that the base code doesn’t violate any of them.

Want me to follow this up with a more compatible eslint config?

**Comment by joostdecock** [comment]
I've re-added eslint to the patterns too. I took it out a while back because it wasn't properly setup (and I seemed to have a hard time getting around to it). So it would be a good occasion to do it across the board (or at least for all patterns/plugins).

Perhaps something that is not obvious: Pretty much all `package.json` files are auto-generated. If you want to update/add a dependency you can change `config/dependencies.yaml` and run `yarn reconfigure`.

This is perhaps a bit hackish, but since so many of our packages are typically used together in 1 project (think freesewing.org) I really wanted to ensure that dependency versions are consistent across the board so we don't end up with multiple copies in the bundle.

Long story short: Yes, I would love your help on this eslint stuff.
Also, we use the rules in `packages/prettier-config` (published as `@freesewing/prettier-config`
